lib/iex/lib/iex/info.ex

```diff
@@ -1,7 +1,7 @@
 defprotocol IEx.Info do
   @fallback_to_any true
 
-  @spec info(term()) :: [{info_name :: String.t(), info :: String.t()}]
+  @spec info(term()) :: [{info_name :: String.Chars.t(), info :: String.t()}]
```

Comment:
Do we have a test where we pass an atom or something of sorts?

Reply:
We don't. We call `to_string()` but we don't test it. Should we?